xmodule/modulestore/store_utilities.py

```diff
@@ -16,8 +16,7 @@ def _prefix_only_url_replace_regex(pattern):
     """
     Match urls in quotes pulling out the fields from pattern
     """
-    return re.compile("""
-        (?x)                      # flags=re.VERBOSE
+    return re.compile("""(?x)     # flags=re.VERBOSE
```

Comment:
Why don't we just use [the`re.VERBOSE` flag](https://docs.python.org/3/library/re.html#re.VERBOSE)? Explicit flags are more readable than the inline ones.

Reply:
I found other code going for this approach, so i opted for consistency. i do agree about making it a explicit flag, so I'll just do that